src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/alerts/AlertsResource.kt

```diff
@@ -137,3 +208,36 @@ data class NomisAudit(
   @Schema(description = "Additional information that is audited")
   val auditAdditionalInfo: String? = null,
 )
+
+@Schema(description = "A request to create an alert in NOMIS")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+data class CreateAlertRequest(
+  @Schema(description = "The alert code")
+  @NotNull
```

Comment:
TODO - forgot how do get decent validation error messages for non-null attributes, this is nonsense but I will come back to it